basic/03a_command_position.py

```diff
@@ -29,7 +29,7 @@
 freq    = freq_hz * 2.0 * pi  # [rad / sec]
 amp     = pi * 0.25           # [rad] (45 degrees)
 
-duration = 10.0               # [sec]
+duration = 4              # [sec]
```

Comment:
@iamtesch @CalZim Does anyone know why this changed to 4 seconds? It appears to be 10 seconds in the [matlab example](https://github.com/HebiRobotics/hebi-matlab-examples/blob/master/basic/x-series_actuator/ex3a_command_position.m#L34)

Reply:
Not sure -- I think this might have just been a testing change.  I'd prefer to keep it matching the MATLAB example for consistency.